pkg/cmd/templates/templates.go

```diff
@@ -49,7 +51,8 @@ Examples:
 {{end}}{{end}}{{ if or $visibleFlags.HasFlags $explicitlyExposedFlags.HasFlags}}
 Options:
 {{ if $visibleFlags.HasFlags}}{{flagsUsages $visibleFlags}}{{end}}{{ if $explicitlyExposedFlags.HasFlags}}{{flagsUsages $explicitlyExposedFlags}}{{end}}{{end}}{{ if .HasSubCommands }}
-Use "{{$rootCmd}} <command> --help" for more information about a given command.{{end}}{{ if $optionsCmdFor}}
+Use "{{$rootCmd}} <command> --help" for more information about a given command.{{end}}
+{{ if $optionsCmdFor}}{{ if $hasContainerOption}}Use "{{$describeCmdFor}} POD" to obtain container name values for that pod.{{end}}
```

Comment:
There are still some unwanted line breaks being generated here. I think, but didn't check, it's because the first `if` of this line must start by the end of the previous line, otherwise you'll have a template line break outside of this if. See `oc -h` (line break between the two "Use..." lines) and `oc get -h` (two line breaks before the "Use..."  line) for examples of what I'm saying.